### PR TITLE
convert directly to spark dataframe from download

### PR DESCRIPTION
**Pull request by clu0**
convert directly from json to pyspark dataframe in download, avoid going through pandas dataframe and dealing with <NA>

### CODE REVIEW

File: cleanlab_studio/internal/api/api.py [hunk 1]
```diff
@@ -155,25 +162,44 @@ def get_label_column_of_project(api_key: str, project_id: str) -> str:
...
 
     :param api_key:
     :param cleanset_id:
     :param all: whether to download all Cleanlab columns or just the clean_label column
-    :return: return (rows, id_column)
+    :return: return a dataframe, either pandas or spark. Type is Any because don't want to require spark installed
     """
     res = requests.get(
         cli_base_url + f"/cleansets/{cleanset_id}/columns?all={all}",
+        params=dict(to_spark=to_spark),
```
Comment (axl1313) [inline cleanlab_studio/internal/api/api.py:181]: nit: can we move the `all` query param to this argument as well instead of adding it to the url manually

**Review by axl1313 — COMMENTED** [review]
code lgtm! just wanted to confirm that you tested that the pandas version still works as expected and that the pyspark version works in databricks? will approve on confirmation

(also have one small nit)

**Comment by clu0** [comment]
> code lgtm! just wanted to confirm that you tested that the pandas version still works as expected and that the pyspark version works in databricks? will approve on confirmation
> 
> (also have one small nit)

yup things still work

File: cleanlab_studio/internal/api/api.py (hunk 1 above)
Comment (clu0) [reply cleanlab_studio/internal/api/api.py]: sounds good updated

**Review by axl1313 — APPROVED** [review]
lgtm!

**Pull request merged**